backend/common/utils/rollup.py

```diff
@@ -32,6 +32,56 @@ def ancestors(cell_type):
     return ancestors
 
 
+def get_valid_descendants(
+    cell_type: str, valid_cell_types: frozenset[str], cell_counts: Optional[dict[str, int]] = None
+):
+    """
+    Get valid descendants for a cell type.
```

Comment:
Adding a bit more description here would be really useful later. This function is used in 2 ways:

1. To get a simple list of descendants for a cell-type _pre-rollup_ with no pruning
2. To identify cell-types to exclude _post-rollup_ per the heuristic by return `[]` for its descendants, which the caller (`are_cell_types_not_redundant_nodes()`) will interpret as a cell-type to exclude post rollup.

I think it would also be useful to state in the docstring to look at `are_cell_types_not_redundant_nodes()` for a description of the heuristic